available_images.md

```diff
@@ -0,0 +1,135 @@
+# Available Deep Learning Containers Images
+
+Once you've selected your desired Deep Learning Containers image, continue with the one of the following:
```

Comment:
This should follow "The following table lists ..."

Reply:
Done